apps/mull-ui/src/app/pages/login/login.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { CustomTextInput } from '@mull/ui-lib';
+import * as Yup from 'yup';
+import { useFormik } from 'formik';
+import logo from '../../../assets/mull-logo.png';
+import { ReactComponent as TwitterIcon } from '../../../assets/icons/login-icons/twitter.svg';
+import { ReactComponent as GoogleIcon } from '../../../assets/icons/login-icons/google.svg';
+import { ReactComponent as FacebookIcon } from '../../../assets/icons/login-icons/facebook.svg';
+
+import './login.scss';
+
+export interface LoginProps {
+  history: History;
```

Comment:
This is not the right History type. You should import just import the history package and it should be fine.

Reply:
`import { History } from 'history';`